test-utils/runtime-tester/src/run_test.rs

```diff
@@ -51,30 +52,32 @@ impl Scenario {
 
             let start_time = Instant::now();
 
-            last_block = env.clients[0].produce_block(block.height).unwrap().unwrap();
+            last_block = env.clients[0]
+                .produce_block(block.height)?
+                .ok_or(Error::Other(String::from("No block has been produced")))?;
             env.process_block(0, last_block.clone(), Provenance::PRODUCED);
 
             block_stats.block_production_time = start_time.elapsed();
 
             runtime_stats.blocks_stats.push(block_stats);
         }
 
-        runtime_stats
+        Ok(runtime_stats)
     }
 }
 
-#[derive(Serialize, Deserialize)]
+#[derive(Serialize, Deserialize, Debug)]
```

Comment:
Debug is needed to print scenario in case of error. It is better to implement Debug as json serialization, so we can run 'cargo fuzz fmt <target> <input> > scenario.json' and have scenario that we can run separately from fuzz.